### DISCUSSION TITLE
Change Link Color in Hello Elementor Theme

### DISCUSSION THREAD

**Discussion by tiffitiff** [elementor/elementor#15599] [Developers Community]
I'm trying to change all the links to my brand colors in the Hello Elementor theme. Right now, all the links are a pink shade. I'm not finding where to change the theme color options in WordPress. Attach is samples of the links I'm referring to. Can anyone help me with this issue? I attached screenshots.

<img width="975" alt="Screen Shot 2021-07-10 at 12 36 13 AM" src="https://user-images.githubusercontent.com/86681260/125151858-469b1c80-e117-11eb-91ee-dc12451bb54e.png">
<img width="1189" alt="Screen Shot 2021-07-10 at 12 37 15 AM" src="https://user-images.githubusercontent.com/86681260/125151859-469b1c80-e117-11eb-905f-88d0e78655af.png">

**Comment by Nazrinn** [comment]
Site settings > Typography, then you should have a section to change your link's color. =)

[Screenshot](https://user-images.githubusercontent.com/36900693/125649957-b98e23f7-15ee-4a2f-b5bd-07680a906f9a.png).
